docker-compose.override.yml

```diff
@@ -6,6 +6,7 @@ services:
     ports:
       - "${DATABASE_PORT:-5432}:${DATABASE_PORT:-5432}"
   caluma:
+    image: projectcaluma/caluma:latest-dev
```

Comment:
`latest` is a specific tag with a special name, other tags might also be the latest but with a different argument set for instance.

Therefore I would simply call this tag `dev`.